library/cohesity_job.py

```diff
@@ -741,6 +741,14 @@ def update_vmware_job(module, job_meta_data, job_details):
         if len(module.params.get('include')) != 0:
             vms = module.params.get('include')
             include_vm_ids = get_vmware_ids(module, job_meta_data, job_details, vms)
+            remove_existing = module.params.get('remove_existing')
```

Comment:
Instead of remove_existing ,can we do replace_existing ?

Reply:
Okay